File: src/main/java/printScreen/TXTHandler.java
package printScreen;

import jdk.jshell.execution.Util;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;

public class TXTHandler {
    public static String content(String filename) {
        InputStream inputStream = TXTHandler.class.getResourceAsStream(filename);
        if (inputStream != null) {
            try (BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream))) {
                StringBuilder stringBuilder = new StringBuilder();
                String line;
                while ((line = reader.readLine()) != null) {
                    stringBuilder.append(line).append("\n");
                }
                return stringBuilder.toString();
            } catch (IOException e) {
                System.out.println("Error reading file: " + e.getMessage());
            }
        }
        return null;
    }
}

File: engine/src/main/kotlin/TXTHandler.kt
import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

object TXTHandler {
    fun content(filename: String): String {
        val inputStream = TXTHandler::class.java.getResourceAsStream(filename)
        if (inputStream != null) {
            try {
                BufferedReader(InputStreamReader(inputStream)).use { reader ->
                    val stringBuilder = StringBuilder()
                    var line: String?
                    while ((reader.readLine().also { line = it }) != null) {
                        stringBuilder.append(line).append("\n")
                    }
                    return stringBuilder.toString()
                }
            } catch (e: IOException) {
                println("Error reading file: " + e.message)
            }
        }
        return ""
    }
}
